iterative/utils/helpers.go

```diff
@@ -44,7 +44,8 @@ func getGHCML(v string) string {
 	return fmt.Sprintf(`sudo mkdir -p /opt/cml/
 sudo curl --location --url %s --output /opt/cml/cml-linux
 sudo chmod +x /opt/cml/cml-linux
-sudo ln -s /opt/cml/cml-linux /usr/bin/cml`, v)
+sudo ln -s /opt/cml/cml-linux /usr/bin/cml
+sudo ln /opt/cml/cml-linux /usr/bin/cml-internal`, v)
```

Comment:
```suggestion
sudo ln -s /opt/cml/cml-linux /usr/bin/cml-internal`, v)
```

Reply:
Not this time. The whole point of this pull request is having two names for the same file.

Reply:
Soft links are ineffective because they get resolved and the problem is the same.

Reply:
8f233c117e9ae088a8822dae26d1cfba61d756c2 — titled _“Add `cml-internal` hard link”_

Reply:
kewl beans

Reply:
```suggestion
sudo ln /opt/cml/cml-linux /usr/bin/cml-internal`, v) # hard link to fix cml#920
```